gradle/libs.versions.toml

```diff
@@ -30,11 +30,12 @@ hilt = "2.51.1"
 hiltNavigationCompose = "1.2.0"
 junit = "4.13.2"
 kotlin = "2.0.0"
+kotlinxSerializationJson = "1.6.3"
 ksp = "2.0.0-1.0.21"
 lifecycle = "2.8.4"
 material3 = "1.2.1"
 media3 = "1.4.1"
-navigation = "2.7.7"
+navigation = "2.8.0-rc01"
```

Comment:
Nit: since `2.8.1` is released, can we use that?

Reply:
Done